includes/admin/payments/class-refund-items-table.php

```diff
@@ -349,6 +349,7 @@ public function single_row( $item ) {
 	public function display() {
 		$singular = $this->_args['singular'];
 
+		wp_nonce_field( 'bulk-' . $this->_args['plural'] );
```

Comment:
@robincornett Since we are adding this manually now lets create a nonce that is specific to the refund action.